webapp/greenhouse.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, job: dict):
         self.employment: str = _get_metadata(job, "employment")
         self.date: str = job["updated_at"]
         self.questions: dict = job.get("questions", {})
-        self.department: str = Department(_get_metadata(job, "department"))
+        self.departments : list = list(map(lambda d: Department(d), _get_metadata(job, "departments") or []))
```

Comment:
Stray space here I think

Reply:
Is it better now @nottrobin ?

Reply:
The space, yes. But the lint check still fails. Are you able to run `black --line-length 79"? If it's still not passing maybe @jkfran could help you out?